test/conditional_logic_SUITE.erl

```diff
@@ -47,7 +47,12 @@ init_per_suite(Config) ->
         sumo:persist(Module, Module:new("John", "Doe", 30)),
         sumo:persist(Module, Module:new("Jane Jr.", "Doe", 5)),
         sumo:persist(Module, Module:new("Joe", "Armstrong")),
-        sumo:persist(Module, Module:new("Alan", "Turing", 102, "Computer St."))
+        sumo:persist(Module, Module:new("Alan", "Turing", 102, "Computer St.")),
+
+        case Module of
+          sumo_test_people_riak -> timer:sleep(5000);
```

Comment:
I'm very aware that you will hate me… even more… after this comment but: can you keep the style and create a `sumo_test_utils:sleep_if_required(Module)` and put this case (in a function-head pattern-matching fashion) in it?
That way we will not need to have cases and conditionals checking module names in the tests themselves.